internal/authorization/browser_linux.go

```diff
@@ -18,6 +18,6 @@ package authorization
 import "os/exec"
 
 func openBrowser(url string) error {
-	cmd := exec.Command("open", url)
+	cmd := exec.Command("xdg-open", url)
```

Comment:
I **guess** that if `open` was used it was tested on some distro; is there any distro that features the `open` command and could break with this change?

If so I'd verify existance and execute the correct command.

Reply:
Hi @fredmaggiowski I've tried to look for info online for the `open` vs `xdg-open` command on Linux distros.
The `xdg-open` command is part of `xdg-utils` by [freedesktop.org](https://www.freedesktop.org/wiki/). 
From their website:
> Freedekstop.org hosts the development of free and open source software, focused on interoperability and shared technology for open-source graphical and desktop systems.

I think is used in most of the distros, and I don't know any distro using `open` command.
Let me know what you think about it.